types/webextension-polyfill/namespaces/devtools_inspectedWindow.d.ts

```diff
@@ -109,7 +104,7 @@ export namespace DevtoolsInspectedWindow {
          * @param options Optional. The options parameter can contain one or more options.
          * @returns A function called when evaluation completes.
          */
-        eval(expression: string, options?: EvalOptionsType): Promise<[any, EvalCallbackExceptionInfoType]>;
```

Comment:
This seems like another change from `any` to `unknown` in an output position.

Reply:
This is one of those reasons I talked about: only the caller knows what type it's supposed to be and should cast it where the call is being made.